recipe/meta.yaml

```diff
@@ -6,17 +6,18 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/TartanLlama/expected/archive/v{{ version }}.tar.gz
+  url: https://github.com/TartanLlama/expected/archive/refs/tags/v{{ version }}.tar.gz
   sha256: 8f5124085a124113e75e3890b4e923e3a4de5b26a973b891b3deb40e19c03cee
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
-    - make  # [unix]
+    - git
+    - ninja
```

Comment:
I think cpp-expected is using fetch_content and git to get the `add-tl.cmake` command.

Let's get add-tl by adding another url block, something like this:
```
   - url: https://github.com/TartanLlama/tl-cmake/tarball/master
     fn: tl-cmake.tar.gz.                                  # Just makes things prettier...
     sha256: de91a9367871ea267d705c626a68c4716f4150c30acb9435915052a1de0f8398
```

Then we'll need to create a patch to disable the fetch in `CMakeLists.txt`.

Reply:
Updated the recipe to fetch tl-cmake from the source section.